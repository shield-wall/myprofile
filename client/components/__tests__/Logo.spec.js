import { RouterLinkStub, shallowMount } from '@vue/test-utils'
import Logo from '../Logo'

function factory () {
  return shallowMount(Logo, {
    stubs: {
      NuxtLink: RouterLinkStub
    },
    mocks: {
      localePath: i => i
    }
  })
}

describe('Logo', () => {
  test('mounts properly', () => {
    const wrapper = factory()
    expect(wrapper.find('div')).toBeTruthy()
  })

  test('renders properly', () => {
    const wrapper = factory()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
