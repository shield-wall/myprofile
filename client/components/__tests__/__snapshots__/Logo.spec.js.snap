// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders properly 1`] = `
<a class="text-3xl font-bold">
  My Profile
</a>
`;
